apis/cluster/serverpool.go

```diff
@@ -34,7 +36,20 @@ type ServerPool struct {
 	Name              string
 	Image             string
 	Size              string
-	BootstrapScript   string
+	BootstrapScripts  []string
 	Subnets           []*Subnet
 	Firewalls         []*Firewall
 }
+
+func BuildServerPoolScript(serverPool *ServerPool) ([]byte, error) {
```

Comment:
Can we decouple this function from the API? Instead of passing in a serverPool can we just pass in a slice of strings?

That way we just have a generic append function?